packages/@aws-cdk/aws-eks-v2-alpha/lib/managed-nodegroup.ts

```diff
@@ -75,6 +75,14 @@ export enum NodegroupAmiType {
    * Amazon Linux 2023 (x86-64)
    */
   AL2023_X86_64_STANDARD = 'AL2023_x86_64_STANDARD',
+  /**
+   * Amazon Linux 2023 Neuron (x86-64)
+   */
+  AL2023_X86_64_NEURON = 'AL2023_x86_64_NEURON',
+  /**
+   * Amazon Linux 2023 with Nvidia-GPU support (x86-64)
+   */
+  AL2023_X86_64_NVIDIA = 'AL2023_x86_64_NVIDIA',
```

Comment:
Unless I'm missing something, those comments seem a little vague, especially calling it GPU "support", given the AMI will not change the hardware. Something like this might be more explicit?

```suggestion
  /**
   * Amazon Linux 2023 with AWS Neuron drivers (x86-64)
   */
  AL2023_X86_64_NEURON = 'AL2023_x86_64_NEURON',
  /**
   * Amazon Linux 2023 with NVIDIA drivers (x86-64)
   */
  AL2023_X86_64_NVIDIA = 'AL2023_x86_64_NVIDIA',
```